client/transaction-pool/graph/src/ready.rs

```diff
@@ -116,6 +125,19 @@ pub struct ReadyTransactions<Hash: hash::Hash + Eq, Ex> {
 	best: BTreeSet<TransactionRef<Hash, Ex>>,
 }
 
+impl<Hash: hash::Hash + Eq, Ex> parity_util_mem::MallocSizeOf for ReadyTransactions<Hash, Ex>
```

Comment:
Would it be possible to fix the derive to allow custom bounds (like in serde?) Cause it's really superfluous and can get out of sync when someone adds a field to the original struct easily.

Reply:
Actually, I was able to remove this custom impl almost everywhere

But custom bound would be useful later, yes, we'll add it in `parity-common`